Added device_id to input/output properties, updates schemas and examp… (#110)
AMWA-TV/is-11: docs/Interoperability.md

In order to prevent unnecessary polling of the Stream Compatibility Management API, changes are signaled via the IS-04 versioning mechanism.

> When properties of any Input/Output are changed, then the `version` attribute of the relevant IS-04 Device MUST be incremented. For this purpose Inputs/Outputs are linked to the corresponding devices via "device_id" property.

Reviewer: Firstly, small suggested change:

```suggestion
When properties of any Input/Output are changed, then the `version` attribute of the relevant IS-04 Device MUST be incremented.
Inputs/Outputs identify the corresponding Devices via the `device_id` property.
```

Reviewer: Secondly, not related to the PR, but a question about the previous sentence "when properties of any Input/Output are changed, ..."

Does "properties" mean formally the `/properties` endpoint, so that whenever e.g. `status` changes, a Device `version` update is required?

Reviewer 2: My understanding is that "properties" refer to the general properties of an Input/Output object referenced by a given id, not only what is being accessible through /properties. This requirement relates to the fact that inputs and outputs are not in the registry such that a Controller would know to check the device input/output state when polling the device version ...

Author: Hi everyone. Sorry for the late response, wasn't working last couple days due to sickness.

Thank you for the proposal, word "identify" fits here better than "is linked", committing this one.
As for the not PR-related question, my understanding pretty much reflects one of Alains: properties are there to track changes of input and output objects without adding them to the registry directly.
